GSoC-2021/Kush-Kothari.md

```diff
@@ -0,0 +1,82 @@
+# GSoC 2021 Expression of Interest
+## Kush Kothari (Web and Python developer | ML Enthusiast)
+#### Email: kushkothari2001@gmail.com
+#### GitHub profile: [kkothari2001](https://github.com/kkothari2001)
+#### LinkedIn profile: [Kush Kothari](https://www.linkedin.com/in/kush-kothari-ba013218b/)
+
+
+## Assessment Step 1.
+
+Link to the deployed site: [ohuploaderkushk](https://ohuploaderkushk.herokuapp.com/)
+
+Link to activity page on OpenHumans: [GSoC assessment for Kush Kothari](https://www.openhumans.org/activity/gsoc-assesment-for-kush-kothari/)
+
+## Assessment Step 2.
+
+### Django Projects:
+- [To-Do App](https://github.com/kkothari2001/django-todo): App that can be used to maintain a list of tasks that can be dynamically added and deleted once completed.
+- [URL Shortner](https://github.com/kkothari2001/django-url-shortner): A bitly clone that stores URLs in a database as a hash and redirects when the URL is clicked.
+
+Please Note: While these projects show my Django skills, I believe they might not be the best way to show some of my other web development skills like web design. I urge you to go through the following links which are a showcase of some of the best projects I have made.
+
+- [The SRA Website](https://sravjti.tech/): This is the website for the Society of Robotics and Automation, VJTI. I am a member of it and have made the entire site using HTML,Sass(CSS) and JavaScript. The code is curently private (because we are working on a backend authentication system right now). But if you wish, I can provide a bundled together zip archive of all the front end mockups in HTML,Sass and JS.
```

Comment:
Thank you - the project and website looks really fun. Is this a static site or does it pull from a database? You don't have to show me the code but if it does link up with a backend (I guess it does if you're working on an authentication system!) then please could you tell me the specific pages where I can see that integration in action?

Reply:
The final siteis  a static site, the site was coded in react but then a React framework, [Next JS](https://nextjs.org/) was used to convert the resulting react SPA into a static site to ensure faster loading times and better SEO as a whole. This site was then hosted using GitHub Pages that builds the site into a static site and loads it on the domain.

Soon, being tech enthusiasts, we plan to add a blogging section that allows members of SRA to add posts relating to tech on the site. However including entire blogs in the site will greatly impact loading times and SEO rankings. So we will include data fetching on this site from a completely different site (that will only be hosted for its backend) . So we're not really linking it to a direct backend. It is currently being hosted by Github Pages